rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, book, :person
```

Comment:
- [OPTIONAL] I think there is a typo here, It should be `:book` with simi-colon instead of `book` 👌